poodle/src/components/index.ts

```diff
@@ -1,2 +1,5 @@
 export { default as ApplyStatus } from './ApplyStatus/ApplyStatus';
-export { Footer as Footer } from './default/Footer'
\ No newline at end of file
+export { Footer as Footer } from './default/Footer';
```

Comment:
```suggestion
export { Footer } from './default/Footer';
```
assertion 안해줘도 될 것 같습니다.

Reply:
아? 그렇네요!
다음부턴 이렇게 안해야겠어요.